criu/include/cr_options.h

```diff
@@ -100,6 +100,22 @@ struct irmap_path_opt {
 	struct irmap *ir;
 };
 
+enum criu_mode {
+	CR_UNSET = 0,
+	CR_DUMP,
+	CR_PRE_DUMP,
+	CR_RESTORE,
+	CR_LAZY_PAGES,
+	CR_CHECK,
+	CR_PAGE_SERVER,
+	CR_SERVICE,
+	CR_SWRK,
+	CR_DEDUP,
+	CR_CPUINFO,
+	CR_EXEC_DEPRECATED,
+	CR_SHOW_DEPRECATED,
+};
```

Comment:
Would it be possible to update `image_dir_mode()` in `crtools.c` with these enum values?

Reply:
Thanks for review! Sure, I forgot about it.